### ISSUE TITLE
_meta missing for non-object responses

### ISSUE THREAD

**Issue by ericdevelops** [webflow/js-webflow-api#13]
Recently I've been attempting to poll Webflow for the `X-RateLimit-Remaining` header to pre-check before attempting to make a series of calls. I began to notice that the _meta item wasn't available in all requests (only seems to work for Webflow.items)

I forked the project to look into this issue and determine if this was a bug or by design. I found that response bodies that parse into a simple array fail to append the rateLimit object returned by buildMeta() because...well you can't append an object to an array. I wouldn't mind submitting a PR but am curious as to whether this fix belongs in the responseHandler and a check for the body type determines how to append the rateLimit object or if this fix really resides with the ResponseWrapper.

**Comment by ericruiz-developer** [comment]
Thought anyone?

**Comment by ericdevelops** [comment]
@nullren 

Yes, I figured converting the arrays to objects would clearly be a breaking change. My PR avoids the breaking changes but obviously has a flaw in that I reference an arbitrary index such as 0. For now, I've forked and implemented my fix since I need this now.

Thanks for the response!

**Comment by rubenmueller** [comment]
@ericdevelops thanks for the quick fix – I ran into this issue today and was looking for a solution. The API rate limits are crazy low, so I need to implement additional logic to delay certain calls if the limit is reached.

I hope the Webflow dev team soon finds some time to dedicate to the API and improve this library and the API's endpoints.

**Comment by johnagan** [comment]
Hello 👋 It looks like this issue has been resolved, so I'm closing it out. If we missed something feel free to re-open 🙇
